docs/tutorial/rln-chat2-live-testnet.md

```diff
@@ -123,3 +123,21 @@ Connecting to storenode: 16Uiu2HAmPLe7Mzm8TsYUubgCAW1aJoeFScxrLj8ppHFivPo97bUZ
 >> /exit
 quitting...
 ```
+
+# Trouble shooting
+
+## compilation error: found possibly newer version of crate
```

Comment:
That is a good idea to have a troubleshooting section, if the `make clean` worked, please update the command in this description, and then we are good to merge the PR.